package.json

```diff
@@ -10,9 +10,14 @@
         "production": "mix --production"
     },
     "devDependencies": {
+        "@tailwindcss/forms": "^0.5.2",
+        "alpinejs": "^3.4.2",
+        "autoprefixer": "^10.4.2",
         "axios": "^0.21",
         "laravel-mix": "^6.0.6",
         "lodash": "^4.17.19",
-        "postcss": "^8.1.14"
+        "postcss": "^8.4.6",
+        "tailwindcss": "^3.1.0",
+        "vite": "^5.2.6"
```

Comment:
Tailwind soll in dem Projekt nicht verwendet werden, sonder Bootstrap. Es muss nachdem Breeze installiert wurde, wieder manuelle entfernt werden.